build/bundle-konflux.Dockerfile

```diff
@@ -48,7 +48,7 @@ LABEL operators.operatorframework.io.metrics.project_layout=go.kubebuilder.io/v3
 LABEL distribution-scope=public
 LABEL release="10.17.1"
 LABEL url="https://docs.openshift.com/container-platform/4.17/windows_containers/index.html"
-LABEL vendor="Red Hat"
+LABEL vendor="Red Hat, Inc."
```

Comment:
Is this a new normal, or just a konflux thing? wondering if we need to adjust otherplaces. eg

https://github.com/openshift/windows-machine-config-operator/blob/master/config/manifests/bases/windows-machine-config-operator.clusterserviceversion.yaml#L27

Reply:
No idea. All I know is that konflux required it.